recipes/versionparsing.jl/scripts/activate.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+# build_base.sh will insert lines defining name, uuid, version, and git_url
+
+pkgsrc="${CONDA_PREFIX}/share/julia/clones/$name.jl"
+
+julia -e"
+using Pkg, UUIDs
+name = \"${name}\"
+uuid = UUID(\"${uuid}\")
+version_str = \"${version}\"
+version = v\"${version}\"
+git_rev = \"v${version}\"
+pkgsrc = \"${pkgsrc}\"
+local_git = joinpath(pkgsrc, \".git\")
+local_git_temp = joinpath(pkgsrc, \"git\")
+remote_git = \"${git_url}\"
+if !haskey(Pkg.dependencies(), uuid)
+    if !isdir(local_git) && isdir(local_git_temp)
+        # Move git to .git
+        mv(local_git_temp, local_git)
+    end
+    if isdir(local_git)
```

Comment:
When is this not true?

Reply:
At the moment this is never true in the current configuration. This is because we are downloading a tarball which does not contain a git repository. I understand this to be the preference of conda-forge. We are not cloning a git repository for this package, which I understand is frowned upon in conda-forge.

The case used for this package is the `isdir(pkgsrc)` situation. In this case we use `Pkg.develop` to integrate the package into the activate project environment. This does not require to the folder to contain a git repository for hashing purposes.

The activate script is meant to be adaptable to for multiple Julia packages with this package serving as the template. 

In the case where we are using a git repository for an unregistered package or an unregistered version of a package, the `git` cases become relevant.

Reply:
I guess we can make an exception for julia packages to use git repostitories.

Reply:
We're using `git` now. Are we good?

Reply:
Thanks. Can you remove the other `if` conditions then?

Reply:
We could. I was thinking about using this as a template though, and we might still need to use these other conditions for packages that are not stored or distributed via `git`. That's actually one potential application of using conda: distributing Julia packages that do not use `git`.

My preference would be to leave this alone as the prototype template reference, but slim down the other packages as we apply this template.

Reply:
I removed the other if conditions.